core/src/main/java/io/opencensus/stats/View.java

```diff
@@ -42,20 +48,43 @@
   public abstract Measure getMeasure();
 
   /**
+   * The {@link Aggregation}s associated with this {@link View}.
+   */
+  public abstract List<Aggregation> getAggregations();
+
+  /**
    * Columns (a.k.a Tag Keys) to match with the associated {@link Measure}.
-   * If no dimensions are specified, then all stats are recorded. Columns must be unique.
    *
-   * <p>Note: The returned list is unmodifiable, attempts to update it will throw an
-   * UnsupportedOperationException.
+   * <p>{@link Measure} will be recorded in a "greedy" way. That is, every view aggregates every
+   * measure. This is similar to doing a GROUPBY on view’s columns. Columns must be unique.
    */
   public abstract List<TagKey> getColumns();
 
   /**
-   * Applies the given match function to the underlying data type.
+   * Returns the time {@link Window} for this {@code View}.
+   *
+   * @return the time {@link Window}.
    */
-  public abstract <T> T match(
-      Function<? super DistributionView, T> p0,
-      Function<? super IntervalView, T> p1);
+  public abstract Window getWindow();
+
+  /**
+   * Constructs a new {@link View}.
+   */
+  public static View create(
+      Name name,
+      String description,
+      Measure measure,
+      List<Aggregation> aggregations,
```

Comment:
A SortedSet sorts the items using a Comparator, so it would be harder for the user to control the order.  Then they would need to match inputs in the SortedSet with outputs in a List.  I think that could also be confusing, since the set and list order their elements in different ways (sorted vs explicit), and users might not expect the order of their elements to match.  Users could also write something like `new TreeSet<Aggregation>(Arrays.asList(Aggregation.StdDev.create(), Aggregation.Count.create()))` and accidentally expect the results to also be in that order.